forge-gui/res/cardsfolder/upcoming/snowborn_simulacra.txt

```diff
@@ -0,0 +1,15 @@
+Name:Snowborn Simulacra
+ManaCost:X U U
+Types:Sorcery
+A:SP$ RepeatEach | ValidTgts$ Permanent.nonToken | TgtPrompt$ Select up to X target nontoken permanents | TargetMin$ X | TargetMax$ X | RepeatSubAbility$ DBConjure | RepeatTargeted$ True | SubAbility$ DBPutFromHand | StackDescription$ SpellDescription | SpellDescription$ Conjure a duplicate of each of X target nontoken permanents into your hand. Those cards perpetually gain "You may spend mana as though it were mana of any color to cast this spell." If X is 5 or more, you may put one of them from your hand onto the battlefield.
+SVar:DBConjure:DB$ MakeCard | DefinedName$ Remembered | Zone$ Hand | RememberMade$ True | ImprintMade$ True | SubAbility$ DBEffect | SubAbility$ DBCleanup
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Futurist Spellthief's Perpetual Effect | Triggers$ Update
```

Comment:
Then here do `RememberObejcts$ Imprinted` and you'll make one effect card in the command zone for all the cards (instead of X different ones)